src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js

```diff
@@ -112,24 +97,23 @@ const ConcurrentMaintenanceStore = {
         })
         .catch((error) => {
           console.log(error);
-          commit('setReadyToRemove', !updatedReadyToRemove);
```

Comment:
This implementation is required, If there is an error message, we need to go back to the current value. Working in vue2

Reply:
updated